distribution/client/src/client.ts

```diff
@@ -43,7 +43,7 @@ export default class Client {
     this.healthChecker = new HealthChecker(process.env.JENKINS_URL || 'http://127.0.0.1:8080');
     this.update = new Update(this.app, { healthChecker: this.healthChecker });
     this.status = new Status(this.app, { flavor: process.env.FLAVOR });
-    this.errorTelemetry = new ErrorTelemetry(this.app, { flavor: process.env.FLAVOR });
+    this.errorTelemetry = new ErrorTelemetry(this.app, this.update, { flavor: process.env.FLAVOR });
```

Comment:
Why not just add this to the options map?  simplifies having to update all the contructors which is why the map is there.

Reply:
I thought about this initially, but no. Does not work as contrary to the flavor, the UL changes all the time. So passing it once at the beginning in the constructor won't fly.